src/blocks/media-block/components/save.js

```diff
@@ -0,0 +1,67 @@
+import { Component } from "@wordpress/element";
+import cls from 'classnames';
+
+export class MediaBlockSaveComponent extends Component{
+
+    render () {
+
+        const { attributes } = this.props;
+
+        const isLinkAvailable = attributes.url !== undefined && attributes.url !== "";
+
+        let titleText = isLinkAvailable ?
+            <a href={ attributes.url } className={"c-block__title-link u-theme--link-hover--dark"}> {attributes.title} </a> :
+            attributes.title;
+
+        const styles = cls(
+            'c-media-block c-block',
+            {'c-block__row': attributes.alignment === "left"},
+            {'c-block__stacked': attributes.alignment === "center"}
+        );
+
+        return (
+            <div className={styles}>
+                <div className={"c-block__image"}>
+                    <div className="c-block__image-outer-wrap">
+                        <div className="c-block__image-wrap">
+                            <img src={ attributes.imageURL } />
+                        </div>
+                    </div>
+                </div>
+                <div className={"c-block__content u-spacing u-color--gray"}>
```

Comment:
Add the following classes: 
` u-border-left--black--at-large u-theme--border-color--darker--left`

Reply:
Depending on if the sidebar is on or off, you will need to add the following classes:

**On:**
`l-grid-item l-grid-item--s--2-col`

**Off:**
`l-grid-item l-grid-item--s--3-col`

Reply:
HI Brent
Could you send me correct button? Because I get html text from https://www.adventist.org/study-the-bible/ and paste into media-block component. So i have green color for this button

Reply:
The correct markup can be [found here](https://alps.adventist.io/v3/?p=atoms-button-outline) but what I'm commenting about above the the color in the backend. I included comments inline below.

Reply:
Thank you